nifty-core/src/main/java/com/facebook/nifty/core/NettyServerTransport.java

```diff
@@ -74,6 +75,8 @@
     private final NettyServerConfig nettyServerConfig;
     private final ChannelStatistics channelStatistics;
 
+    private AtomicReference<SSLServerConfiguration> sslConfiguration;
```

Comment:
updateSSLConfiguration might be called on any thread. So i didnt want the reference to get swapped out underneath when we were creating a handler.